assets/src/dashboard/app/views/myStories/header/test/header.js

```diff
@@ -137,6 +137,33 @@ describe('My Stories <Header />', function () {
   });
 
   it('should have 3 toggle buttons, one for each status that say how many items belong to that status', function () {
+    const { getByText, queryByText } = renderWithProviders(
+      <LayoutProvider>
+        <Header
+          filter={STORY_STATUSES[0]}
+          stories={fakeStories}
+          search={{ keyword: '', setKeyword: jest.fn() }}
+          sort={{ value: STORY_SORT_OPTIONS.NAME, set: jest.fn() }}
+          totalStoriesByStatus={{
+            all: 19,
+            draft: 9,
+            [STORY_STATUS.PUBLISHED_AND_FUTURE]: 10,
+          }}
+          view={{
+            style: VIEW_STYLE.GRID,
+            pageSize: { width: 200, height: 300 },
+          }}
+          wpListURL="fakeurltoWordPressList.com"
+        />
+      </LayoutProvider>
+    );
+    expect(getByText('All Stories (19)')).toBeInTheDocument();
+    expect(getByText('Drafts (9)')).toBeInTheDocument();
+    expect(getByText('Published (10)')).toBeInTheDocument();
+    expect(queryByText('Private')).not.toBeInTheDocument();
```

Comment:
Can we also have a test for when `canReadPrivatePosts: false` and make sure that Private does not display.